jwst/jump/jump.py

```diff
@@ -31,7 +31,7 @@ def detect_jumps (input_model, gain_model, readnoise_model,
     image.  Also, a 2-dimensional read noise array with appropriate values for
     each pixel is passed to the detection methods.
     """
-    if max_cores is None:
+    if max_cores is 'none' or None:
         numslices = 1
     else:
         num_cores = multiprocessing.cpu_count()
```

Comment:
Maybe it would be more useful and less confusing to the user to move the log message (in the next line) to the end of the block where numslices is known and then change to message to include both available cores and actual number in use, e.g. "Found <n> cores; using <m>".

Reply:
I agree, I wanted to change this as well it threw me off. It does eventually say 'Creating N processes for jump detection', I'll combine those into a single message.